php-web-form/README.md

```diff
@@ -0,0 +1,20 @@
+# SalesLoft API - PHP Web Form Handler
+
+This guide will look at how a PHP web form might be handled to add a person to the SalesLoft
+platform, and then add the newly created person to a specific inbound cadence. PHP is chosen
+as it is a very common language for marketing sites to be implemented in.
```

Comment:
I don't know if you need the justification of why we wrote this guide in PHP. Unless you're trying not to scare other developers who may want to work at salesloft

Reply:
I think it's worth it because sales people will know that their marketing site can be connected most likely